pynblint/repository.py

```diff
@@ -67,6 +70,50 @@ def large_file_paths(self) -> List[Path]:
                     large_files.append(file_path)
         return large_files
 
+    def _get_dependencies(self) -> set:
+        dependencies = set()
+        for root, dirs, files in os.walk(self.path):
+            for f in files:
+                # check if exist a requiremets.txt file
+                if f.endswith("requirements.txt"):
```

Comment:
This recursive search is not necessary. `requirements.txt` – as well as other files in which dependency declaration typically occurs – is expected to be placed at the root of a Python project.